api/v1/models.go

```diff
@@ -274,8 +275,10 @@ type NamespaceEnvelope struct {
 }
 
 type VulnerabilityEnvelope struct {
-	Vulnerability *Vulnerability `json:"Vulnerability,omitempty"`
-	Error         *Error         `json:"Error,omitempty"`
+	Vulnerability   *Vulnerability   `json:"Vulnerability,omitempty"`
+	Vulnerabilities *[]Vulnerability `json:"Vulnerabilities,omitempty"`
+	NextPage        *string          `json:"NextPage,omitempty"`
```

Comment:
I'm pretty sure this doesn't need to be a pointer. If the string is `""` it should also be excluded via `omitempty`. I'd double check, though.